.github/workflows/helm_charts_scalar.yml

```diff
@@ -41,31 +41,31 @@ jobs:
           helm repo add scalar https://scalar-labs.github.io/helm-charts
 
       - name: Set up chart-testing
-        uses: helm/chart-testing-action@v2.0.1
+        uses: helm/chart-testing-action@v2
 
       - name: Run chart-testing (lint)
         run: ct lint --config .github/ct.yaml
 
   kubeaudit-chart:
     runs-on: ubuntu-latest
     env:
-      KUBE_AUDIT_VERSION: 0.16.0
+      KUBE_AUDIT_VERSION: 0.22.0
```

Comment:
It seems that there is no `supported version` explicitly. So, I just chose the latest version.
https://github.com/Shopify/kubeaudit